networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/subnets/NodeIdToDataColumnSidecarSubnetsCalculator.java

```diff
@@ -59,18 +41,18 @@ static NodeIdToDataColumnSidecarSubnetsCalculator create(
             .get()
             .flatMap(
                 slot -> {
-                  final SpecVersion specVersion = spec.atSlot(slot);
-                  final NodeIdToDataColumnSidecarSubnetsCalculator calculatorAtSlot;
-                  if (specVersion.getMilestone().isGreaterThanOrEqualTo(SpecMilestone.FULU)) {
-                    calculatorAtSlot =
-                        createAtSlot(
-                            SpecConfigFulu.required(specVersion.getConfig()),
-                            specVersion.miscHelpers(),
-                            slot);
-                  } else {
-                    calculatorAtSlot = NOOP;
+                  SpecVersion version = spec.atSlot(slot);
+                  if (version.getMilestone().isGreaterThanOrEqualTo(SpecMilestone.FULU)) {
```

Comment:
I think this stuff should be exposed via `SpecLogic` and we should remove those "if milestone is fulu".
But this can be for another PR